src/resources/elements/primeDesignSystem/prange-slider/prange-slider.ts

```diff
@@ -2,29 +2,26 @@ import { bindable } from "aurelia-typed-observable-plugin";
 import { bindingMode, computedFrom, customElement } from "aurelia-framework";
 import "./prange-slider.scss";
 import { Utils } from "../../../../services/utils";
-import { NumberService } from "../../../../services/NumberService";
-import tippy, { Instance } from "tippy.js";
+import { toBigNumberJs } from "../../../../services/BigNumberService";
+import { BigNumber } from "ethers";
```

Comment:
Is there a way to remove these `BigNumber` packages from `<prange-slider>`?
Those seems too specific for a general p-system component

Reply:
I think is ok.  the component for now only supports bignumbers, which is all we need.